src/components/Flex/index.test.tsx

```diff
@@ -4,13 +4,13 @@ import { render } from '@testing-library/react';
 import FlexItem from 'components/FlexItem';
 import Flex from '.';
 
-const hasAllClasses = (container, classes) =>
-  classes.every((c) => Object.values(container.firstChild.classList).includes(c));
+const hasAllClasses = (container: HTMLElement, classes: string[]) =>
+  classes.every((c) => container?.firstElementChild?.classList.contains(c));
```

Comment:
nullish coalescing, definitely want to see more of this ❤️

Reply:
to be fair, nullish coalescing is `??`, this would be optional chaining, but i agree, i love me some optional chaining, and also nullishg coalescing.